go/mysql/connection.go

```diff
@@ -49,12 +49,16 @@ func (this *ConnectionConfig) Equals(other *ConnectionConfig) bool {
 	return this.Key.Equals(&other.Key) || this.ImpliedKey.Equals(other.ImpliedKey)
 }
 
-func (this *ConnectionConfig) GetDBUri(databaseName string) string {
+func (this *ConnectionConfig) GetDBUri(databaseName string, includeMultibyte bool) string {
 	hostname := this.Key.Hostname
 	var ip = net.ParseIP(hostname)
 	if (ip != nil) && (ip.To4() == nil) {
 		// Wrap IPv6 literals in square brackets
 		hostname = fmt.Sprintf("[%s]", hostname)
 	}
-	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?interpolateParams=true&autocommit=true&charset=utf8mb4,utf8,latin1", this.User, this.Password, hostname, this.Key.Port, databaseName)
+	var multibyteCharset string
+	if includeMultibyte {
+		multibyteCharset = ",gbk,gb2312,big5,cp932,sjis"
```

Comment:
is `multibyteCharset` a correct description? `utf8` is also a multibyte charset. Is there a better definition?

Reply:
How about `riskCharset` and `includeRiskCharset`?

Reply:
sounds good.